tools/ci_build/github/apple/default_training_ios_framework_build_settings.json

```diff
@@ -14,6 +14,8 @@
         "--use_xcode",
         "--enable_training_apis",
         "--build_apple_framework",
+        "--use_coreml",
+        "--use_xnnpack",
```

Comment:
@baijumeswani @askhade adding these to the training package build as they might be useful for inference. let me know if that's ok.